lib/stripe.js

```diff
@@ -24,6 +24,9 @@ Stripe.USER_AGENT_SERIALIZED = null;
 var APP_INFO_PROPERTIES = ['name', 'version', 'url'];
 
 var exec = require('child_process').exec;
+var util = require('util');
+var EventEmitter = require('events').EventEmitter;
+var objectAssign = require('object-assign');
```

Comment:
Sorry for the nit, but can we alphabetize this list? Maybe put the capitalized vars on top.